src/vswhere.lib/vswhere.lib.vcxproj

```diff
@@ -59,9 +59,7 @@
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>WIN32;_DEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <SDLCheck>true</SDLCheck>
-      <ControlFlowGuard>Guard</ControlFlowGuard>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
-      <AdditionalOptions>/Zi %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Did you check that you can build in VS okay? While your changes are fine for MSBuild, I found, in practice, that some options can't be moved out into props or VS (more specifically, the VC project system) doesn't know how to show them e.g., for possible modification, in the VS properties window.

Reply:
I didn't test locally, will pull and verify

Reply:
Build will no doubt work. I was more concerned with opening the project properties in VS. I'm not saying these definitely will have a problem, but I was surprised by some similar changes that completely break the ability to change properties in the UI for what seems like basic MSBuild property overrides.

Reply:
I'll sign off, but please do check this. If it works, great. Good to go. Even if it doesn't, maybe it's okay? I would say: if you can make the same change using the VS property sheets UI (or did you already), whatever it does is perfect.

Reply:
Ah I see, thanks. The property pages opens fine and loads from the common props. If you make changes, the changes are saved directly to the vcxproj and overwrite what is in the common props.

While testing locally I did find conflicting settings in Debug builds for the test project. Pushing a fix for that.

Let me know if this doesn't cover your questions.